presto-product-tests/README.md

```diff
@@ -1,125 +1,106 @@
 # Presto product tests
 
-## Test configuration
+## Installing dependencies
 
-Test assume that you have hadoop and Presto cluster running. 
+*Running product test requires at least 4GB free memory to be available.*
 
-To make it work you need either:
- - define environment variables for hadoop ${HADOOP_MASTER} and Presto ${PRESTO_MASTER} with their IP numbers
- - create test-configuration-local.yaml with following (example content):
+#### On GNU/Linux
+* [```docker >= 1.10```](https://docs.docker.com/installation/#installation)
 
-```
-databases:
-  hive:
-    host: 172.16.2.10
-
-  presto:
-    host: 192.168.205.1
-```
-
-## Running tests
+    ```
+    wget -qO- https://get.docker.com/ | sh
+    ```
 
-Product tests are not run by default. To start them use run following command:
+* [```docker-compose >= 1.60```](https://docs.docker.com/compose/install/)
 
-```
-java -jar target/presto-product-tests-*-executable.jar --config-local tempto-configuration-local.yaml
-```
+    ```
+    pip install docker-compose
+    ```
 
-## Running tests with Using preconfigured docker based clusters
+#### On Mac OS X
 
-In case you do not have an access to hadoop cluster or do not want to spent your time on configuration, this section
-describes how to run product tests with preconfigured docker based clusters.
+* [```docker-toolbox >= 1.10```](https://www.docker.com/products/docker-toolbox)
 
-### Prerequisites
+On Mac OS X installing docker-toolbox gives access to preconfigured bash environment
+with ```docker``` and ```docker-compose``` available. To start this bash environment
+select "Docker Quickstart Terminal" from Launchpad. Note that all commands given in
+further parts of this docs should be run from this environment.
```

Comment:
Just a sanity-check-note, but please renumber that steps covered in this PR are only going to work with all the changes introduced in its other commits. So - in order to actually run this you must either use this branch or rebase all four commits to your branch.
This will of course not be the case after integrating this PR to master ;-)